packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -171,7 +171,7 @@ fn.currentVersion.addAlias('live');
 
 ## Layers
 
-The `lambda.LayerVersion` class can be used to define Lambda layers and manage
+The `lambda.LayerVersion` class can be used to define Lambda layer versions and manage
```

Comment:
Revert this change and add a statement at the end of this section about configuring removal policies.

Reply:
I added better documentation to the README. Let me know if it's too much detail